_episodes/format.md

```diff
@@ -3,51 +3,260 @@ title: "File format"
 teaching: 0
 exercises: 0
 questions:
-- "Key question (FIXME)"
+- "What is a file format?"
+- "What file formats should I use?"
 objectives:
-- "First learning objective. (FIXME)"
+- "Name common formats used in climate-related domains."
+- "Understand different types of data."
+- "Choose correct file formats for your work."
+
 keypoints:
-- "First key point. Brief Answer to questions. (FIXME)"
+- "Choose formats that are common to your field/community to ensure the interoperability and reusability of your data."
+- "Make sure that the file formats you choose can hold the necessary data elements and information."
+- "Decide on how long do you intend to preserve your data."
+- "Make sure to check requirements of the repository where data is stored."
 ---
 
+## File format
+
+A [file format](https://en.wikipedia.org/wiki/File_format) is the structure of a file.
+It determines how the data within the file is organized.
+A File format is also called a file extension.
+Files are usually named as `filename.format`.
+For example, climate.jpeg is a file named climate with [jpeg](https://en.wikipedia.org/wiki/JPEG) format.
+You might know that `jpeg` is a commonly used format for digital images.
+Therefore, the file `climate` is an image.
```

Comment:
```suggestion
Therefore, the file `climate` is probably an image.
```
See the second paragraph under 'usage' [here](https://en.wikipedia.org/wiki/Filename_extension#Usage), it states that on unix systems usually this suffix is ignored when reading a file. So it's mostly intended for the user of the file. But for many files on linux, and can probably change or remove the extension, and the right programs can still read the file.

Reply:
Agree, I added a callout to explain this.